app/Services/WorklogService.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Services;
+
+use App\Constants\FlashMessages;
+use App\Exceptions\Worklogs\WorklogNotCreatedException;
+use App\Exceptions\Worklogs\WorklogNotFoundException;
+use App\Exceptions\Worklogs\WorklogNotUpdatedException;
+use App\Models\Worklog;
+use Carbon\Carbon;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Support\Facades\Auth;
+
+class WorklogService
+{
+    /**
+     * @return mixed
+     * @throws WorklogNotFoundException
+     */
+    public function getLoggedInUserWorklogs(): Collection
+    {
+        $worklogs = Worklog::where('user_id', Auth::id())
+            ->orderBy('created_at', 'desc')
+            ->get();
+        if(!$worklogs)
+        {
+            throw new WorklogNotFoundException('No worklogs created yet. Click Add Task to create new worklog');
+        }
```

Comment:
instead of throwing exception. we can show message "No worklogs created yet. Click Add Task to create new worklog" on ui

Reply:
Yes, the exception message thrown from here is being rendered in the UI via the controller within the catch block.